quickstarts/contrib.md

```diff
@@ -0,0 +1,45 @@
+# Contributing to the Gemini API Cookbook
```

Comment:
Isn't this sort of information usually put at the top level in a `CONTRIBUTING.md` file?

Reply:
Oh.. it exists.

https://github.com/google-gemini/cookbook/blob/main/CONTRIBUTING.md